command-line/keyboard-shortcuts.md

```diff
@@ -53,6 +53,16 @@ You don't have to use keyboard shortcuts for everything on your computer, but af
 
 - [Windows Keyboard Shortcuts Cheat Sheet](https://code.visualstudio.com/shortcuts/keyboard-shortcuts-windows.pdf)
 
+**Mac Utilities (optional):**
+
+* **Alfred 3** - Alfred is an award-winning app for macOS which boosts 
+             your efficiency with hotkeys, keywords, text expansion and more.
+             Search your Mac and the web, and be more productive with custom 
+             actions to control your Mac.
+             
+* **Magnet 2.0** - Window manager for Mac.
+
+More Apps: [Mac-Utilities](https://www.producthunt.com/ask/7567-what-are-your-must-have-mac-apps-utilities)
```

Comment:
Please describe what this resource is a little more.

Reply:
I think window manager for mac is good description. But if you want to be more descriptive then replace it with **Magnet keeps your workspace organized. Activated by dragging, customizable keyboard shortcuts or via menu bar, Magnet declutters your screen by snapping windows into organized tiles.**

Reply:
I mean line 65

Reply:
What changes he has to do in line 65 now ?